.github/workflows/gradle.yml

```
@@ -855,158 +863,207 @@ jobs:
+    env:
+      HOME: /actions-runner/_work/airbyte/airbyte
+      GITHUB_WORKSPACE: /actions-runner/_work/airbyte/airbyte
+    # github:
+    #   workspace: /home/runner
+
+    steps:
+      - name: Fix EC-2 Runner
+        run: |
+          mkdir -p /actions-runner/_work/airbyte/airbyte && mkdir -p /actions-runner/_work/airbyte/airbyte/.kube
+        
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Delete default old docker and replace it with a new one
+        shell: bash
+        run: |
+          sudo apt-get remove docker.io || sudo apt-get remove docker
+          curl -fsSL https://get.docker.com | bash -
+
+      - name: Cache Build Artifacts
+        uses: ./.github/actions/cache-build-artifacts
+        with:
+          cache-key: ${{ secrets.CACHE_VERSION }}
+          cache-python: "false"
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9' 
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "17"
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "lts/gallium"
+
+      - name: Install unzip
+        shell: bash
+        run: sudo apt-get update && sudo apt-get install -y unzip
+
+      - uses: azure/setup-helm@v3
+        with:
+          version: 'latest'
+          token: ${{ secrets.GITHUB_TOKEN }}
+        id: install
+
+      - uses: azure/setup-kubectl@v3
+        with:
+          version: 'v1.21.2' # default is latest stable
+
+      - name: Install tmate and it's dependencies
+        if: inputs.debug_mode == 'true'
+        shell: bash
+        run: |
+          sudo rm /var/lib/dpkg/lock
+          sudo rm /var/lib/dpkg/lock-frontend
+          sudo apt-get -y install tmate
+          sudo rm -f /var/lib/dpkg/lock
+          sudo rm -f /var/lib/dpkg/lock-frontend
+      
+      - name: Start tmate session in background
+        if: inputs.debug_mode == 'true'
+        shell: bash
+        run: |  
+          tmate -S /tmp/tmate.sock new-session -d               # Launch tmate in a headless mode
+          tmate -S /tmp/tmate.sock wait tmate-ready             # Blocks until the SSH connection is established
+          tmate -S /tmp/tmate.sock display -p '#{tmate_ssh}'    # Prints the SSH connection string
+          tmate -S /tmp/tmate.sock display -p '#{tmate_ssh_ro}' # Prints the read-only SSH connection string
+          tmate -S /tmp/tmate.sock display -p '#{tmate_web}'    # Prints the web connection string
+          tmate -S /tmp/tmate.sock display -p '#{tmate_web_ro}' # Prints the read-only web connection string
+
+      - name: Start a local k8s cluster
+        uses: jupyterhub/action-k3s-helm@v3
+        with:
+          k3s-version: v1.21.2+k3s1
+          helm-version: v3.9.4
+          extra-setup-args: --docker --kube-apiserver-arg service-node-port-range=6000-32767
+
+      - name: Try the cluster !
+        run: kubectl get pods -A
```

Can we do more to inform the user about this failure?
```suggestion
        run: kubectl get pods -A || echo "failed to get pods!"
```